developer_docs/feature/excel-export-html-table.md

```
@@ -0,0 +1,160 @@
+                 class="ui-button-success mr-1"
+                 actionListener="#{myController.downloadExcel}"
+                 ajax="false" />
+```
+
+`ajax="false"` is mandatory — without it the browser will not receive the file download.
+
+---
+
+## Excel file structure
+
+```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)